### DISCUSSION TITLE
Heatmap convert dataframe

### DISCUSSION THREAD

**Discussion by anderMartin92** [kernc/backtesting.py#708] [Q&A] [answered]
Hello, when I optimize the strategy I would like to be able to use the data returned by heatmap. In the image it looks like the last value has no column and is the result of my function (maximize = optim_func). It simply returns Return/DD.
<img width="1000" alt="Captura de Pantalla 2022-08-02 a las 2 06 21" src="https://user-images.githubusercontent.com/62348899/182266103-c4573493-7eac-4d40-87e2-74fff3b0847f.png">

I have tried to convert heatmap to dataframe but I lose the last column. What I want is to access that data to keep the values > 3.
<img width="343" alt="Captura de Pantalla 2022-08-02 a las 2 17 04" src="https://user-images.githubusercontent.com/62348899/182266081-0b4432dc-fa9b-4bf8-8b1d-d7e93f5fb06e.png">

Thank you very much

**Comment by kernc** [comment, marked as answer]
Returned heatmap is a series. Use [`Series.to_frame()`](https://pandas.pydata.org/docs/reference/api/pandas.Series.to_frame.html):
```py
df = heatmap.to_frame()
```

**Reply by anderMartin92** [reply]
thank you bro!
